src/App.tsx

```diff
@@ -8,16 +8,10 @@ function App() {
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>
-          Edit <code>src/App.tsx</code> and save to reload.
+          I love cats. My favourite cat is Hello Kitty!
```

Comment:
I think India's cat would be offended

Reply:
Good point Hannah, thanks. I have updated it now

Reply:
Okay thanks